apps/automation_crews/financial_crew/Readme.md

```diff
@@ -0,0 +1,11 @@
+### Start ONLY "Financial Crew"
+#### Requires ./src/fincancial_crew/assets/Bank-of-America-Bank-Statement-TemplateLab.com_.pdf to exist
+#### fix some imports
```

Comment:
is this a todo for the future?

Reply:
No, it was a comment added to the description. But now I have deleted it.